libs/live-network/src/network.ts

```diff
@@ -158,8 +162,52 @@ const extractErrorMessage = (raw: string): string | undefined => {
   return;
 };
 
+export type LiveNetworkRequest<T> = {
+  url?: string;
+  method?: "GET" | "POST" | "PUT" | "DELETE";
+  baseURL?: string;
+  params?: unknown;
+  data?: T;
+  timeout?: number;
+};
+export type LiveNetworkResponse<T> = {
+  data: T;
+  status: number;
+};
+export const newImplementation = async <T = unknown, U = unknown>(
+  request: LiveNetworkRequest<U>,
+): Promise<LiveNetworkResponse<T>> => {
+  let response: AxiosResponse<T>;
+
+  if (!("method" in request)) {
+    request.method = "GET";
+  }
+
+  if (request.method === "GET") {
+    if (!("timeout" in request)) {
+      request.timeout = getEnv("GET_CALLS_TIMEOUT");
+    }
+
+    response = await retry(() => axios(request), {
+      maxRetry: getEnv("GET_CALLS_RETRY"),
```

Comment:
Do you think it'd make sense to also have `maxRetry` this in `LiveNetworkRequest` ?

Reply:
What bothering me, is that this is not part of the `live-network` package.
I think we should do, but maybe with a small refacto.